js/main.js

```diff
@@ -0,0 +1,412 @@
+      row.forEach((el) => {
+        if (
+          el[1] === targetBtnName
+            && (specialBtn === undefined
+              || specialBtn === 'space'
+              || specialBtn === 'tab'
+              || specialBtn === 'enter')
+        ) {
+          symbol = symbolChoise(el);
+        }
+      });
+    });
+
+    if (specialBtn === 'tab') {
+      symbol = '  ';
+    }
+
+    if (specialBtn === 'enter') {
+      symbol = '\n';
+    }
+
+    // Adding symbol to textArea
+    textArea.setRangeText(symbol, textArea.selectionStart, textArea.selectionEnd, 'end');
+
+    if (specialBtn === 'backspace') {
+      if (textArea.selectionStart > 0) {
+        const pos = textArea.selectionStart;
+        textArea.value = textArea.value.slice(0, pos - 1) + textArea.value.slice(pos, textArea.value.length);
+        textArea.setRangeText('', pos - 1, pos - 1, 'end');
+      }
+    }
+
+    if (specialBtn === 'del') {
+      const pos = textArea.selectionStart;
+      if (textArea.selectionStart <= textArea.value.length) {
+        textArea.value = textArea.value.slice(0, pos) + textArea.value.slice(pos + 1, textArea.value.length);
+        textArea.setRangeText('', pos, pos, 'end');
+      }
+    }
+
+    const specialBtnEl = document.querySelector(`.${specialBtn}`);
+
+    if (
+      specialBtn === 'shift-left'
+        || specialBtn === 'shift-right'
+        || specialBtn === 'capslock'
+    ) {
+      if (shiftPress === false) {
+        addActiveState(specialBtnEl);
+        caseUp();
+      } else {
+        removeActiveState(specialBtnEl);
+        caseDown();
+      }
+    }
+
+    if (specialBtn === 'arrow') {
+      const pos = textArea.selectionStart;
+
+      switch (targetBtnName) {
+        case 'ArrowUp': setCaretPosition(textArea, pos - 69);
+          break;
+        case 'ArrowRight': setCaretPosition(textArea, pos + 1);
+          break;
+        case 'ArrowDown': setCaretPosition(textArea, pos + 69);
+          break;
+        case 'ArrowLeft':
+          if (textArea.selectionStart > 0) {
+            setCaretPosition(textArea, pos - 1);
+          }
+          break;
+        default:
+          break;
+      }
+    }
+
+    if (specialBtn === 'alt-left') {
+      altLeftPress = activeBtnHighlighting(altLeftPress, specialBtnEl);
+    }
+
+    if (specialBtn === 'alt-right') {
+      altRightPress = activeBtnHighlighting(altRightPress, specialBtnEl);
+    }
+
+    if (specialBtn === 'ctrl-left') {
+      ctrlLeftPress = activeBtnHighlighting(ctrlLeftPress, specialBtnEl);
+    }
+
+    if (specialBtn === 'ctrl-right') {
+      ctrlRightPress = activeBtnHighlighting(ctrlRightPress, specialBtnEl);
+    }
+  }
+
+  textArea.focus();
+}
+
+document.addEventListener('keydown', shiftUpKeyboard);
+keyboard.addEventListener('click', printingInTextArea);
+document.addEventListener('keyup', shiftDownKeyboard);
+document.addEventListener('keydown', (evt) => {
+  if (evt.shiftKey && evt.altKey) pageLangChanging();
```

Comment:
Please use one of construction below. It's not good practise to use if without block brackets

`evt.shiftKey && evt.altKey && pageLangChanging();`
or 
```
if (evt.shiftKey && evt.altKey) {
   pageLangChanging();
}

```